apple-app-site-association.json

```diff
@@ -1,5 +1,5 @@
 {
-  "activitycontinuation": {
+  "activity ntinuation": {
```

Comment:
This seems like an accidental deletion of some characters.

Reply:
Aww good catch!